pkg/detectors/jdbc/sqlite.go

```diff
@@ -14,12 +14,18 @@ type sqliteJDBC struct {
 	testing  bool
 }
 
-func (s *sqliteJDBC) ping(ctx context.Context) bool {
+var cannotVerifySqliteError error = errors.New("sqlite credentials cannot be verified")
```

Comment:
Not something for this PR, but I wonder if we could verify the sqlite credentials if the DB was a local file?